prog/vm/host_nexus.rb

```diff
@@ -245,8 +245,22 @@ def get_boot_id
   end
 
   def available?
-    sshable.cmd("true")
-    true
+    ssh_session = sshable.start_fresh_session
+    is_available = false
+
+    begin
+      checks = [
+        -> { vm_host.check_storage_smartctl(ssh_session, vm_host.disk_devices) },
+        -> { vm_host.check_storage_nvme(ssh_session, vm_host.disk_devices) },
+        -> { vm_host.check_storage_read_write(ssh_session, vm_host.disk_devices) },
+        -> { vm_host.check_storage_kernel_logs(ssh_session, vm_host.disk_devices) }
+      ]
```

Comment:
overall, we do a lot of checks and raise a flag when anything fails. This works, but in case of a failure, the on-call engineer will need to spend some time to understand what was wrong. Maybe emitting logs about what failed inside the check_ functions can be useful.

Reply:
you're right. I added logs to checkers